CHANGELOG.md

```diff
@@ -4,6 +4,21 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
+## [1.5.10] - 2022-02-08
```

Comment:
Also:

https://github.com/PyTorchLightning/pytorch-lightning/pull/11672 b8e360dafa77fa9da397159391d3f46be149cc9a [CLI] Fix bug that forces overriding `configure_optimizers`

Reply:
can you cherry-pick this? I was getting failures.

Reply:
Okay. It's was only a problem in master. I've updated the milestone